tests/integ/modin/series/test_apply.py

```diff
@@ -482,17 +480,6 @@ def f(x) -> float:
     eval_snowpark_pandas_result(snow_series, native_series, lambda x: x.map(f))
 
 
-@sql_count_checker(query_count=3)
-def test_apply_convert_dtype(caplog):
-    snow_series = pd.Series([1])
-
-    caplog.clear()
-    WarningMessage.printed_warnings.clear()
-    with caplog.at_level(logging.WARNING):
-        snow_series.apply(lambda x: x, convert_dtype=True)
```

Comment:
Do we want to keep a test around to enforce that no warning is raised, and that the result dtype might differ from pandas?

Reply:
I don't think it makes a lot of sense to add a test to enforce that no warning is raised. If we do the same everywhere else, we'll end up with so many test :) Also, since the parameter is deprecated now, I think it's not worth investing much more into it.